doc/langref.html.in

```diff
@@ -1878,6 +1902,15 @@ or
 
       {#see_also|Sentinel-Terminated Pointers|Sentinel-Terminated Slices#}
       {#header_close#}
+
+      {#header_open|Destructuring Arrays#}
+      <p>
+        Arrays can be destructured:
+      </p>
+      {#code|destructuring_arrays.zig#}
```

Comment:
I don't see a `destructuring_arrays.zig` in this PR or in the existing code.  Did you forget to add it?